bndtools.core/src/bndtools/editor/BndEditor.java

```diff
@@ -598,8 +602,7 @@ public void done(IJobChangeEvent event) {
 
     private Promise<Workspace> loadEditModel() throws Exception {
         // Create the bnd edit model and workspace
-        Workspace ws = Central.getWorkspaceIfPresent();
-        Project bndProject = Run.createRun(ws, inputFile);
+        Project bndProject = LaunchUtils.createRun(inputResource, Mode.EDIT);
```

Comment:
It seems like there is a scenario where `this.inputResource` could be null. `Run.createRun` can handle a null `inputFile` but `LaunchUtils.createRun` cannot handle a null `inputResource`.

Reply:
Is it even possible to open an editor without a resource?